reformat_gherkin/formatter.py

```diff
@@ -268,6 +263,25 @@ def __find_nodes_with_newline(self) -> Set[Node]:
 
         return nodes_with_newline
 
+    def __find_max_step_keyword_width(self) -> int:
+        """
+        Find the length of the longest step keyword in the document. This is
+        used for aligning step keywords.
+        """
+        if self.step_keyword_alignment == AlignmentMode.NONE:
+            # We don't need to align step keywords in this case.
+            return 0
+
+        step_keyword_widths = [
+            get_display_width(node.keyword.strip())
+            for node in self.ast
+            if isinstance(node, Step)
+        ]
+        if step_keyword_widths:
+            return max(step_keyword_widths)
+        else:
+            return 0
```

Comment:
These lines can be refactored to

```python
if ...:
    return ...

return 0
```

Reply:
Ok, will do.

Reply:
Actually, on second thoughts, if we are going to do an early return it is probably better the other way round:

```python
        if not step_keyword_widths:
            return 0
   
        return max(step_keyword_widths)
```

That way both of the early returns in the function are for the sad paths, and the final return is for the happy path, which I think is clearer.